manifests/init.pp

```diff
@@ -527,6 +527,10 @@
 #                                     disable in case CA is delegated to a separate instance
 #                                     type:boolean
 #
+# $server_puppetserver_vardir::       The path of the puppetserver var dir
+#                                     Defaults to '/opt/puppetlabs/server/data/puppetserver'
```

Comment:
Given we have multiple defaults depending on the version, I think we should leave out this line